addons/multiplegap/src/presenter.js

```diff
@@ -1594,17 +1594,49 @@ function Addonmultiplegap_create(){
     MultipleGapKeyboardController.prototype.enter = function (event) {
         window.KeyboardController.prototype.enter.call(this, event);
         if(this.keyboardNavigationElementsLen > 1) {
-            var voicesArray = [];
-            for(var i  = 1; i <this.keyboardNavigationElementsLen; i++) {
-                var $element = this.getTarget(this.keyboardNavigationElements[i]);
-                voicesArray = voicesArray.concat(presenter.getTextVoicesFromPlaceholder($element));
+            if (presenter.configuration.sourceType === presenter.SOURCE_TYPES.AUDIO) {
+                var $element = presenter.$view.find('.placeholder.keyboard_navigation_active_element');
+                if ($element.length == 0) {
+                    readAllElements();
+                } else {
+                    var draggableItemValues = $element.attr('draggableitem').split('-');
```

Comment:
Widziałbym zawartość tego else w funkcji - żeby wiedzieć jaki to przypadek (readSingleElement?).